crates/simulacrum/src/store.rs

```diff
@@ -409,3 +409,194 @@ impl KeyStore {
         self.account_keys.iter()
     }
 }
+
+#[async_trait::async_trait]
```

Comment:
Can we have this be sync vs async? Given this likely wont use a remote store, sync makes things a bit easier/nicer.

Reply:
Having this be async also is likely impossible with how the existing move execution environment works.

Reply:
Fix coming up shortly

Reply:
Addressed here https://github.com/MystenLabs/sui/pull/14350